front-end/dashboard.js

```diff
@@ -51,6 +52,31 @@ new Vue({
                 // do nothing.
             }
         },
+        cursorHide() {
+            let cursorTimer;
+            document.body.addEventListener(
+                'mousemove',
+                function(e) {
```

Comment:
```suggestion
                event => {
```
ES6 short function, no need for `.bind(this)`. Also personally I don't like variables like `e`, even if it's common use :innocent: